iModelCore/ECDb/ECDb/ECSql/SelectStatementExp.h

```diff
@@ -178,6 +178,11 @@ struct LimitOffsetExp final : Exp
 struct OrderBySpecExp final : Exp
     {
     public:
+        enum class NullsOrder {
+            First,
+            Last,
+            NotSpecified

```

Comment:
Better put the NotSpecified value first, so it's the default

Reply:
fixed the order